src/main/resources/static/css/common.css

```diff
@@ -0,0 +1,307 @@
+
+input::-ms-clear{display: none;}
+button{cursor: pointer;}
+button::-moz-focus-inner {padding:0;border:0;}
+a, a:visited{text-decoration: none;}
+a:hover{text-decoration: none;}
+ul li{list-style: none;}
+img{vertical-align: top;}
+
+h1,h2,h3,h4,h5,h6{font-size:inherit;font-weight: 400;}
+
+/* nunito-regular - latin */
+@font-face {
+  font-family: 'Nunito';
+  font-style: normal;
+  font-weight: 400;
+  src: url('../fonts/nunito/nunito-v16-latin-regular.eot'); /* IE9 Compat Modes */
+  src: local(''),
+  url('../fonts/nunito/nunito-v16-latin-regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */ url('../fonts/nunito/nunito-v16-latin-regular.woff2') format('woff2'), /* Super Modern Browsers */ url('../fonts/nunito/nunito-v16-latin-regular.woff') format('woff'), /* Modern Browsers */ url('../fonts/nunito/nunito-v16-latin-regular.ttf') format('truetype'), /* Safari, Android, iOS */ url('../fonts/nunito/nunito-v16-latin-regular.svg#Nunito') format('svg'); /* Legacy iOS */
+}
+
+/* nunito-700 - latin */
+@font-face {
+  font-family: 'Nunito';
+  font-style: normal;
+  font-weight: 700;
+  src: url('../fonts/nunito/nunito-v16-latin-700.eot'); /* IE9 Compat Modes */
+  src: local(''),
+  url('../fonts/nunito/nunito-v16-latin-700.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */ url('../fonts/nunito/nunito-v16-latin-700.woff2') format('woff2'), /* Super Modern Browsers */ url('../fonts/nunito/nunito-v16-latin-700.woff') format('woff'), /* Modern Browsers */ url('../fonts/nunito/nunito-v16-latin-700.ttf') format('truetype'), /* Safari, Android, iOS */ url('../fonts/nunito/nunito-v16-latin-700.svg#Nunito') format('svg'); /* Legacy iOS */
+}
+
+/* nunito-900 - latin */
+@font-face {
+  font-family: 'Nunito';
+  font-style: normal;
+  font-weight: 900;
+  src: url('../fonts/nunito/nunito-v16-latin-900.eot'); /* IE9 Compat Modes */
+  src: local(''),
+  url('../fonts/nunito/nunito-v16-latin-900.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */ url('../fonts/nunito/nunito-v16-latin-900.woff2') format('woff2'), /* Super Modern Browsers */ url('../fonts/nunito/nunito-v16-latin-900.woff') format('woff'), /* Modern Browsers */ url('../fonts/nunito/nunito-v16-latin-900.ttf') format('truetype'), /* Safari, Android, iOS */ url('../fonts/nunito/nunito-v16-latin-900.svg#Nunito') format('svg'); /* Legacy iOS */
+}
+
+/* base */
+
+body{
+  font-family: 'Nunito', serif;
+  font-style: normal;
+  font-weight: 400;
+  color: #fff;
+  font-size: 18px;
+  line-height: 1.4;
+  /*min-width: 1920px;*/
+}
+
+.wrapper{
+  min-height: 100%;
+  overflow: hidden;
+  background: url("../img/background.jpg") no-repeat fixed center;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  padding: 0 2em;
+}
+
+.wrapper-home{
+  background: url("../img/background_home.jpg") no-repeat fixed center;
+  background-size: cover;
+}
+
+.container{
+  max-width: 96.7em;
+  margin: 0 auto;
+}
+
+.sub-container{
+  /*max-width: 90.45em;*/
```

Comment:
@AlexKievDeveloper , Are comments here ok?

Reply:
because frontend in progress I leave this comment to me. I have css in two places, so i don't want to delete every time this comment before push. I will delete this comment when I end front, I promise :)

Reply:
Actually Greengrer is right. You can leave it now but please remove all comments before your next PR. Thanks.